### PR TITLE
add example: DnD 5e ability scores and modifiers

### PR DESCRIPTION
**Pull request by Cube707**
This example shows how to calculate values from frontmatter properties and store them in memory, than reusing and formatting them where needed.

### CODE REVIEW

File: exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+VIEW[ floor(({CON}-10)/2) ][math(hidden):memory^CON_mod]
+```
+```meta-bind
+VIEW[ floor(({INT}-10)/2) ][math(hidden):memory^INT_mod]
+```
+```meta-bind
+VIEW[ floor(({CHR}-10)/2) ][math(hidden):memory^CHR_mod]
+```
+
+- STR: `VIEW[**{STR}**][text(renderMarkdown,class(score))]` (`VIEW[ concat(isPositive({memory^STR_mod})?'+':'', string({memory^STR_mod}))][math(class(moddifyer))]`)
```
Comment (mProjectsCode) [inline exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md:33]: you add a CSS class called `moddifyer`. Shouldn't it be called `modifier` and you didn't include any CSS for it.

Comment (Cube707) [reply exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md:33]: ah sorry, both are correct. just leftover from by own document. I also style the elements, but that didn't seem super relevant to the example

File: exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This example calculates a DnD character's ability modifiers from the ability scores and displays it all in a list.
+
+The modifiers are stored in `memory` and not in the frontmatter. This reduces clutter but requires some hidden fields that calculate the values while the fields in the list just format the value to always contain a `+/-` sign.
+
+
+## Character ability scores
+
+```meta-bind
+VIEW[ floor(({STR}-10)/2) ][math(hidden):memory^STR_mod]
```
Comment (mProjectsCode) [inline exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md:18]: i would do the spacing like this `VIEW[floor(({STR} - 10) / 2)]`, for better readability

File: exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+VIEW[ floor(({INT}-10)/2) ][math(hidden):memory^INT_mod]
+```
+```meta-bind
+VIEW[ floor(({WIS}-10)/2) ][math(hidden):memory^WIS_mod]
+```
+```meta-bind
+VIEW[ floor(({CHR}-10)/2) ][math(hidden):memory^CHR_mod]
+```
+
+- STR: `VIEW[**{STR}**][text(renderMarkdown)]` (`VIEW[ concat(isPositive({memory^STR_mod})?'+':'', string({memory^STR_mod}))][math]`)
```
Comment (mProjectsCode) [inline exampleVault/Advanced Examples/DnD 5e Ability Scores and Modifiers.md:36]: same thing about spacing here `VIEW[concat(isPositive({memory^STR_mod}) ? '+' : '', string({memory^STR_mod}))]`

**Review by mProjectsCode — COMMENTED** [review]
Looks good, except some minor spacing gripes that I have

**Comment by mProjectsCode** [comment]
thanks

**Pull request merged**